cmd/integration/integration.go

```diff
@@ -205,7 +205,7 @@ func startComponents(manifestURL, apiVersion string) (string, string) {
 	scheduler.New(schedulerConfig).Run()
 
 	endpoints := service.NewEndpointController(cl)
-	go util.Forever(func() { endpoints.SyncServiceEndpoints() }, time.Second*10)
+	go util.Forever(func() { endpoints.SyncServiceEndpoints() }, time.Second*4)
```

Comment:
nit: Can you add some comment why do you set to 4 seconds (and why 10 seconds is too long?)